DEPRECATED.md

```diff
@@ -4,198 +4,3 @@
 
 Relying on the `<base>` tag has been deprecated in Contao 5.0 and will no longer work in Contao 6. Use absolute paths
 for links and assets instead.
-
-## kernel.packages
-
-The `kernel.packages` parameter has been deprecated in Contao 4.5 and will be removed in Contao 5.0. Use the
-`Composer\InstalledVersions` class instead.
-
-```php
-$coreVersion = InstalledVersions::getPrettyVersion('contao/core-bundle');
-```
-
-## TL_ASSETS_URL and TL_FILES_URL
-
-The constants `TL_ASSETS_URL` and `TL_FILES_URL` have been deprecated in Contao 4.5 and will be removed in Contao 5.0.
-Use the assets or files context instead:
-
-```php
-// Old syntax
-echo TL_ASSETS_URL;
-echo TL_FILES_URL;
-
-// New syntax
-$container = System::getContainer();
-echo $container->get('contao.assets.assets_context')->getStaticUrl();
-echo $container->get('contao.assets.files_context')->getStaticUrl();
-```
```

Comment:
This needs to be moved to the *UPGRADE.md* file then I think.

Reply:
Done in 348b908366de0479df75bb888e10893566bf831d.